envoy-control-source-consul/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/consul/services/ConsulClusterStateChangesTest.kt

```diff
@@ -91,6 +97,26 @@ class ConsulClusterStateChangesTest {
             .verify()
     }
 
+    @Test
+    fun `should produce event with only matching services`() {
```

Comment:
can u enhance this test in the manner of given when then section as it is done in example in ConsulServiceMapperTest so I would like to have: 

`
// given
var someVar = 1

// when
callToAction(someVar)

// then
assertThatSomethingHasBeenDone() // in ur sample StepVerifer
`